src/material-experimental/mdc-slider/slider-adapter.ts

```diff
@@ -6,133 +6,179 @@
-    throw Error('Method not implemented.');
+  focusInput = (thumb: Thumb): void => {
+    this._delegate._getInputElement(thumb).focus();
   }
-  removeThumbStyleProperty(propertyName: string, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  isInputFocused = (thumb: Thumb): boolean => {
+    return this._delegate._getInput(thumb)._isFocused();
   }
-  setTrackActiveStyleProperty(propertyName: string, value: string): void {
-    throw Error('Method not implemented.');
+  getThumbKnobWidth = (thumb: Thumb): number => {
+    // TODO(wagnermaciel): Check if this causes issues for SSR
+    // once the mdc-slider is added back to the kitchen sink SSR app.
+    return this._delegate._getKnobElement(thumb).getBoundingClientRect().width;
   }
-  removeTrackActiveStyleProperty(propertyName: string): void {
-    throw Error('Method not implemented.');
+  getThumbBoundingClientRect = (thumb: Thumb): ClientRect => {
+    return this._delegate._getThumbElement(thumb).getBoundingClientRect();
   }
-  setValueIndicatorText(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  getBoundingClientRect = (): ClientRect => {
+    return this._delegate._elementRef.nativeElement.getBoundingClientRect();
   }
-  getValueToAriaValueTextFn(): ((value: number) => string) | null {
-    throw Error('Method not implemented.');
+  isRTL = (): boolean => {
+    // TODO(wagnermaciel): Actually implementing this.
+    return false;
   }
-  updateTickMarks(tickMarks: TickMark[]): void {
-    throw Error('Method not implemented.');
+  setThumbStyleProperty = (propertyName: string, value: string, thumb: Thumb): void => {
+    this._delegate._getThumbElement(thumb).style.setProperty(propertyName, value);
   }
-  setPointerCapture(pointerId: number): void {
-    throw Error('Method not implemented.');
+  removeThumbStyleProperty = (propertyName: string, thumb: Thumb): void => {
+    this._delegate._getThumbElement(thumb).style.removeProperty(propertyName);
   }
-  emitChangeEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  setTrackActiveStyleProperty = (propertyName: string, value: string): void => {
+    this._delegate._trackActive.nativeElement.style.setProperty(propertyName, value);
   }
-  emitInputEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  removeTrackActiveStyleProperty = (propertyName: string): void => {
+    this._delegate._trackActive.nativeElement.style.removeProperty(propertyName);
   }
-  emitDragStartEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  setValueIndicatorText = (value: number, thumb: Thumb): void => {
+    this._delegate._setValueIndicatorText(value, thumb);
   }
-  emitDragEndEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  getValueToAriaValueTextFn = (): ((value: number) => string) | null => {
```

Comment:
Did we ever figure out why these have to be arrow functions instead of methods?

Reply:
Yes, MDC is clobbering them. I left a comment here explaining
https://github.com/angular/components/pull/21844/files#diff-f0898cf25f96d9c72cf31d7289a09209c713131e3268f7c96c27b35cc4be537bR39